bootstrap/src/basis/map_intf.ml

```diff
@@ -413,6 +410,18 @@ module type S_ord = sig
   (** [partitioni_map ~f t] partitions [t] into two maps for which [~f] returns
       [First v2] vs [Second v3].  Θ(n) time complexity. *)
 
+  val min_elm: cmp:(('k * 'v) -> ('k * 'v) -> Cmp.t) -> ('k, 'v, 'cmp) t
+    -> ('k * 'v) option
+  (** [min_elm ~f t] iterates from left to right over [t] and returns [Some
+      (k, v)] for the first element which always compares as [Cmp.Lt] or
+      [Cmp.Eq], or [None] if [t] is empty. *)
+
+  val max_elm: cmp:(('k * 'v) -> ('k * 'v) -> Cmp.t) -> ('k, 'v, 'cmp) t
+    -> ('k * 'v) option
+  (** [max_elm ~f t] iterates from left to right over [t] and returns [Some
+      (k, v)] for the first element which compares as [Cmp.Eq] or [Cmp.Gt], or
```

Comment:
```suggestion
      (k, v)] for the first element which always compares as [Cmp.Eq] or [Cmp.Gt], or
```
Just to keep it consistent with `min_elm`. Improves meaning. Guessing this suggestion exceeds line width.

Reply:
Alternatively
```first element which always compares as ...``` to ```first element with compares as [Cmp.Eq] or [Cmp.Gt] to all other elements```
No strong preference here. Just might not be super clear what the element is actually being compared with.

Reply:
How about `leftmost element which compares as [Cmp.Eq] or [Cmp.Gt] to all other elements`?

Reply:
I changed all instances of this documentation to `leftmost element which always compares as [Cmp.Eq] or [Cmp.Gt]`.